app/src/main/java/org/oppia/android/app/player/state/StatePlayerRecyclerViewAssembler.kt

```diff
@@ -887,11 +887,13 @@ class StatePlayerRecyclerViewAssembler private constructor(
     private val interactionViewModelFactoryMap: Map<String, InteractionItemFactory>,
     private val backgroundCoroutineDispatcher: CoroutineDispatcher,
     private val resourceHandler: AppLanguageResourceHandler,
-    private val translationController: TranslationController
+    private val translationController: TranslationController,
+    private val adapterBuilderFactory: BindableAdapter.MultiTypeBuilder.Factory,
+    private val singleAdapterFactory: BindableAdapter.SingleTypeBuilder.Factory
   ) {
-    private val adapterBuilder = BindableAdapter.MultiTypeBuilder.newBuilder(
-      StateItemViewModel::viewType
-    )
+
+    private var adapterBuilder: BindableAdapter.MultiTypeBuilder<StateItemViewModel,
```

Comment:
Can this just be a ``val``? It doesn't seem to need to change.

Reply:
Noted.